pip/cmdoptions.py

```diff
@@ -523,6 +523,56 @@ def only_binary():
 )
 
 
+def strong_hashes():
```

Comment:
Fair enough. Perhaps it'd be useful to have a comment here in the event someone new to pip comes along hoping to score an easy PR and thinks this should just be a constant instead of a function.